src/NanoTranscriber.cpp

```diff
@@ -15,6 +15,9 @@ void distributeMessages(iarc7_msgs::Nano nano_info,
                         ros::Publisher& long_range_pub,
                         ros::Publisher& battery_publisher){
 
+    // From here on out, each of the "offset" variables is multiplied by 1000 to convert 
```

Comment:
You mean 10000?  And tens of microseconds?

Reply:
They should be 1000, that was a mistake

Reply:
Yes, there was a small modification in the firmware that I was unaware of